graphql/end_to_end_test.go

```diff
@@ -29,72 +28,101 @@ type Slow struct {
 func TestPathError(t *testing.T) {
 	schema := schemabuilder.NewSchema()
 
-	type Inner struct{}
+	type enumType int32
```

Comment:
would you put your enum tests in a new TestEnum function? TestPathError is explicitly for testing PathErrors.